wiki/markdown/subscription-with-REST-POST-notification.md

```diff
@@ -1,21 +1,22 @@
 # Subscription with REST POST notification
 
-A valid subscription for REST POST notification is provided below with comments
-for each field.
+Valid subscriptions for REST POST notification are provided below with 
+comments for each field.
 
 _**OBS! Comments are only for documentation purposes and a subscription should
 not contain them. The subscription will be rejected at this moment if it
 contains comments like below.**_
 
-_**Subscription templates can be found [here](https://github.com/eiffel-community/eiffel-intelligence/tree/master/src/main/resources/templates).**_
+**Subscription templates can be found [here](https://github.com/eiffel-community/eiffel-intelligence/tree/master/src/main/resources/templates).**
 
     {
-        // The name of the subscription to make it easy to search for it.
+        // The name of the subscription to make it easy to search for.
         // Only numbers, letters and underscore allowed.
         "subscriptionName" : "Subscription1",
 
-        // The name of the logged in user creating or updating the subscription
-        // added by Eiffel Intelligence if LDAP is enabled. Defaults to an empty string.
+        // The name of the logged in user creating or updating the subscription.
+        // Added by Eiffel Intelligence if LDAP is enabled. Not required. 
+        // Defaults to an empty string.
```

Comment:
I was also hesitating about how to handle this... I will move out the comments from outside the json and make a note on each subscription field before adding the complete json example. Hopefully this can improve the readability.